src/langsmith/deploy-self-hosted-full-platform.mdx

```diff
@@ -75,7 +75,45 @@ As of v0.12.0, the `langgraphPlatform` option is deprecated. Use `config.deploym
       enabled: true
       langgraphPlatformLicenseKey: "YOUR_LANGGRAPH_PLATFORM_LICENSE_KEY"
 ```
-4. In your `values.yaml` file, configure the `hostBackendImage` and `operatorImage` options (if you need to mirror images)
+4. In your `values.yaml` file, configure the `hostBackendImage` and `operatorImage` options (if you need to mirror images). If you are using a private container registry that requires authentication, you must also configure `imagePullSecrets`:
+
+   <Accordion title="Configure authentication for private registries (optional)">
```

Comment:
The `<Accordion>` was hard for me to find and/or know that it was clickable. My preference is to have it all out here, not hidden.

Reply:
Agree, I've taken out and made this a separate, optional section at the end of the doc and linked to it from item 4